main.go

```diff
@@ -64,8 +65,8 @@ type customDNSProviderConfig struct {
 	// These fields will be set by users in the
 	// `issuer.spec.acme.dns01.providers.webhook.config` field.
 
-	//Email           string `json:"email"`
-	//APIKeySecretRef v1alpha1.SecretKeySelector `json:"apiKeySecretRef"`
+	Email           string `json:"email"`
+	APIKeySecretRef certmanagerv1.SecretKeySelector `json:"apiKeySecretRef"`
```

Comment:
Not all of these fields are required by all DNS provider types, hence keeping the contents here generic. Can you revert these changes, and if you feel there could be additional clarification as to what a user should do here then add comments to explain it?

Reply:
I don't think that I included them or maybe this is because they were defined within one of the example of the cert manager project. So, do you suggest that we remove the fields `Email` and `APIKeySecretRef` ?